Kubernetes/helmcharts/azure-policy-addon-arc-connected-clusters/templates/azure-policy-serviceaccount.yaml

```diff
@@ -27,4 +27,4 @@ subjects:
 roleRef:
   kind: ClusterRole
   name: policy-agent
-  apiGroup: rbac.authorization.k8s.io
+  apiGroup: rbac.authorization.k8s.io
```

Comment:
Same here, keep the newline

Reply:
YAML install worked fine without the new line. Why is it a requirement?

Reply:
Newlines help keep the code tidy. As far as I'm aware it's a kubernetes norm to leave a newline at the end of yaml files

Reply:
FYI - Checked some of the files in official helm chart repo. I do not see a newline as a convention being followed.
https://github.com/helm/charts/blob/master/stable/bookstack/templates/rolebinding.yaml